react/src/utils.tsx

```diff
@@ -1,4 +1,8 @@
-import React, { Component, ComponentProps, ComponentType } from 'react'
+import React, { ComponentProps, ComponentType } from 'react'
+
+// XXX: export type { VariantProps } from '@stitches/react'
```

Comment:
should we just delete this? also, I think I'm confused on whether the stitches `VariantProps` is working or causing issues. but down to merge this and see if it fixes the linting errors

Reply:
sgtm!